plugins/source/alicloud/client/spec.go

```diff
@@ -49,3 +54,38 @@ func (s *Spec) Validate() error {
 	}
 	return nil
 }
+
+func (Spec) JSONSchemaExtend(sc *jsonschema.Schema) {
+	one := uint64(1)
+
+	accounts := *sc.Properties.Value("accounts").OneOf[0]
+	accounts.MinItems = &one
+	sc.Properties.Set("accounts", &accounts)
+
+	sc.Required = append(sc.Required, "accounts")
```

Comment:
add to annotation instead of extend

Reply:
Addressed in https://github.com/cloudquery/cloudquery/pull/16355/commits/baa62595a3f268025cc062b4d5900fe069e4e3bd